example/lib/page_remote_config.dart

```diff
@@ -14,9 +14,9 @@ class RemoteConfigPage extends StatefulWidget {
 }
 
 class _RemoteConfigPageState extends State<RemoteConfigPage> {
+  String rcKey = 'testKey';
```

Comment:
does this value change? if it doesn't, use const

Reply:
it does